article/_posts/2020-07-30-DADS.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: review
+title: "Dynamics-Aware Unsupervised Discovery of Skills"
+tags: reinforcement unsupervised
+author: "Antoine Théberge"
+cite:
+    authors: "Archit Sharma, Shixiang Gu, Sergey Levine, Vikash Kumar, Karol Hausman"
+    title:   "Dynamics-Aware Unsupervised Discovery of Skills"
+    venue:   "ICLR 2020"
+pdf: "https://arxiv.org/pdf/1907.01657.pdf"
+---
+
+
+# Highlights
+
+- Unsupervised "skill" discovery without extrinsic rewards
+- Planning in "skill" space allows to generalize to unseen tasks
+
+# Introduction
+
+Model-based reinforcement learning (MBRL) tries to learn the dynamics of the environment as well as a policy that plans the optimal course of action using the learned dynamics. This allows the policy to be learned in more sample-efficient way as well as making able to generalize to new tasks.
+
+However, generalizing to new state distributions remains an open problem, as exploring the full state-space and learning the full dynamics for high-dimensional systems is generally infeasible. The authors want to retain the flexibility of MBRL by using the simplicity of model-free RL to learn low-level behaviors that can be leveraged at test time.  
+
+# Methods
+
+The authors want to learn the dynamics for a large and diverse set of skills, which allows them to use model-based planning in the _skill space_ for solving new tasks.
+
+## Unsupervised skill discovery
+
+Using the common MDP $$M = (S, A, p, r)$$, they want to learn a skill-conditioned policy $$\pi(a \vert s,z)$$ with $$z \in \mathbb(Z)$$ that can be sampled from a prior $$p(z)$$. To plan in the _skill-space_, the authors also propose to learn the dynamics for skills $$q_{\theta}(s'{\vert}s,z)$$, which will be used to plan at test time.
```

Comment:
Although not very important, I'd stick to the same convention when mentioning the _skill space_: i.e. with or without a dash, but always with the same form.

Reply:
Addressed in 4fc7473.